tests/unit/traditional.js

```diff
@@ -30,5 +32,7 @@ exports.testInitializationMultiple = DomUtil.createDomTest(
         test.deepEqual($input.selectivity('value'), [3, 4]);
 
         test.deepEqual($('#selectivity-input').selectivity('value'), [3, 4]);
+
+        test.equal($input.find('select[name="my_select"] option[selected]').length, 2);
```

Comment:
Um, could you give me a hand here? I tried to use `$input.find('select[name="my_select"]').val()` that normally returns a string value/array of strings, but got `undefined` instead. I tried also `$input.find('select[name="my_select"] option[selected]').first` and `$input.find('select[name="my_select"] option[selected]')[0]` but also got `undefined` in both cases.